packages/botonic-nlu/src/preprocessing.js

```diff
@@ -106,3 +106,55 @@ export function detectLang(input, langs) {
   }
   return res
 }
+
+export function parseUtterance(utterance) {
+  let capturedGroup = utterance.match(new RegExp(ENTITIES_REGEX, 'g')) || []
```

Comment:
create constant with new RegExp(ENTITIES_REGEX, 'g') to avoid recreating it

Reply:
rename capturedGroup to capturedGroups?

Reply:
all done!

Reply:
done!